kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/index.vue

```diff
@@ -77,6 +77,7 @@
         </KPageContainer>
       </KGridItem>
     </KGrid>
+    <router-view />
```

Comment:
Hi! Yes, there are some motivations for keeping this in this way. The main motivation is to keep these side panels in sync with the same pattern that will be used in quizzes that will be introduced in this PR https://github.com/learningequality/kolibri/pull/12819, and in that PR you can find more motivations for this like:

> * Removing unnecessary indirection around when/how/where components define what to do re: back and close behavior
> * Providing a better example for handling side panels w/ routing for Lessons
> * Much easier to put things into the Side panel title section

At the end its a tradeof of how much do we want to abstract and reuse, and how much flexibility do we need to avoid making the pattern more complex to follow.

Reply:
Elaborating on the side panel title thing - basically, if the child-component rendered in the `router-view` has it's own `SidePanelModal` then it has the direct ability to set the title prop (or pass to the header slot).

Separately, this way also associates the child-component directly with the route, so the parent component doesn't have to manage a `v-if=""` for when to show the side panel. The route decides if it's shown automatically without our having to deal adding a conditional in the parent.